packtools/sps/validation/aff.py

```diff
@@ -401,9 +401,9 @@ def validate_comparison(self, main_aff: dict):
             got = result["got"]
             items = result["items"]
             if self.affiliation.get("id"):
-                advice = f"Low similarity detected. Review and adjust affiliation ({self.affiliation.get('id')}) to better match the main text."
+                advice = f"The affiliation {self.affiliation.get("id")} is inconsistent with {main_aff}. Check if {self.affiliation.get("id")} is translation of {main_aff}"
```

Comment:
@Rossi-Luciano ajuste as aspas do início e do fim
f'The affiliation {self.affiliation.get("id")} is inconsistent with {main_aff}. Check if {self.affiliation.get("id")} is translation of {main_aff}'

Reply:
OK, @robertatakenaka